docs/labs/flockercontainers.rst

```diff
@@ -0,0 +1,97 @@
+.. _flocker-containers:
+
+=============================
+Running Flocker in Containers
+=============================
+
+For some environments, such as CoreOS, and ECS you must install Flocker inside a container.
+
+.. warning::
+   Installing Flocker in containers is useful for environments such as CoreOS, or Amazon ECS, but please note that doing so is currently experimental.
+
+You can find the relevant Docker Hub images here:
+
+* `Dataset Agent <https://hub.docker.com/r/clusterhq/flocker-dataset-agent/>`_
+* `Container Agent <https://hub.docker.com/r/clusterhq/flocker-container-agent/>`_
+* `Control Service <https://hub.docker.com/r/clusterhq/flocker-control-service/>`_
+* `Docker Plugin <https://hub.docker.com/r/clusterhq/flocker-docker-plugin/>`_
+
+Before you install Flocker in containers, all authentication must be completed.
```

Comment:
"all authentication must be completed" is a bit of a weird phrase. I suggest dropping it and saying "Before you install Flocker in containers, you must have generated the cluster certificate..."